compose/seadoc-standalone.yml

```diff
@@ -0,0 +1,32 @@
+---
+services:
+
+  seadoc:
+    image: ${SEADOC_IMAGE:-seafileltd/sdoc-server:1.0-latest}
```

Comment:
I believe it makes sense to pin this tag (i.e`1.0.3`) to prevent unintentional upgrades. We pin all the other tags as well.

Reply:
Done